src/components/accessories/laboratory/Exams.tsx

```diff
@@ -85,16 +110,35 @@ export const Exams: FC = () => {
   const errorMessage = useSelector((state: IState) =>
     state.laboratories.searchLabs.error?.message
       ? state.laboratories.searchLabs.error?.message
+      : t("common.somethingwrong") ||
+        state.laboratories.updateLab.error?.message
+      ? state.laboratories.updateLab.error?.message
```

Comment:
Please use a different variable in order to mantein the code as clean as possibile.

Plus, add the message inside the mock when the api fails (now seems to be empty).